getting-started/recipe-development-environment.md

```diff
@@ -111,6 +115,11 @@ dependencies {
         <artifactId>rewrite-java-11</artifactId>
         <scope>runtime</scope>
     </dependency>
+    <dependency>
+        <groupId>org.openrewrite</groupId>
+        <artifactId>rewrite-java-17</artifactId>
+        <scope>runtime</scope>
+    </dependency>
```

Comment:
I'm pretty sure we just forgot to add a `java 17` section here. Let me know if I misunderstood something, though.